mm2src/mm2_main/tests/mm2_tests/tendermint_tests.rs

```diff
@@ -112,14 +113,45 @@ fn test_tendermint_withdraw() {
     println!("Send raw tx {}", json::to_string(&send_raw_tx).unwrap());
 }
 
+#[test]
+fn test_tendermint_ibc_withdraw() {
+    const IBC_SOURCE_CHANNEL: &str = "channel-81";
+    const IBC_TARGET_ADDRESS: &str = "cosmos1r5v5srda7xfth3hn2s26txvrcrntldjumt8mhl";
+    const MY_ADDRESS: &str = "iaa1e0rx87mdj79zejewuc4jg7ql9ud2286g2us8f2";
+
+    let coins = json!([iris_testnet_conf(), iris_nimda_testnet_conf()]);
+    let platform_coin = coins[0]["coin"].as_str().unwrap();
+    let token = coins[1]["coin"].as_str().unwrap();
+
+    let conf = Mm2TestConf::seednode(IRIS_TEST_SEED, &coins);
+    let mm = MarketMakerIt::start(conf.conf, conf.rpc_password, None).unwrap();
+
+    let activation_res = block_on(enable_tendermint(&mm, platform_coin, &[], IRIS_TESTNET_RPC_URLS, false));
+    println!("Activation with assets {}", json::to_string(&activation_res).unwrap());
+
+    let activation_res = block_on(enable_tendermint_token(&mm, token));
+    println!("Token activation {}", json::to_string(&activation_res).unwrap());
+
+    let tx_details = block_on(ibc_withdraw(&mm, IBC_SOURCE_CHANNEL, token, IBC_TARGET_ADDRESS, "0.1"));
+    println!("IBC transfer to atom address {}", json::to_string(&tx_details).unwrap());
+
+    let expected_spent: BigDecimal = "0.1".parse().unwrap();
+    assert_eq!(tx_details.spent_by_me, expected_spent);
+
+    assert_eq!(tx_details.to, vec![IBC_TARGET_ADDRESS.to_owned()]);
+    assert_eq!(tx_details.from, vec![MY_ADDRESS.to_owned()]);
+
+    let send_raw_tx = block_on(send_raw_transaction(&mm, token, &tx_details.tx_hex));
```

Comment:
You can add some additional checks to this test to make sure that the transaction does what it's supposed to do, e.g. you can compare the balances of the sender and the receiver before/after the `send_raw_transaction`. I see that the PR is merged now so you can do it on the next PR.

Reply:
the relayer I used in this test doesn't work properly on testnet, that's why I couldn't compare balances. So either we have to deploy relayer for testnet that works properly, or use nucleus chain for this specific test because it works without any issue

Reply:
Sorry, didn't know you were reviewing this. I merged it since it got the minimum of 2 approvals and was under review for a long time. From now on we should be adding only 2 reviewers on a PR (1 is also enough for a small PR), if someone doesn't plan to review a PR, they can remove themselves and another team member can review it instead :)

Reply:
Yeah no problem, I only had this one comment anyways.